app/sigverify.go

```diff
@@ -38,6 +45,73 @@ func SigVerificationGasConsumer(
 		return nil
 
 	default:
-		return canto.SigVerificationGasConsumer(meter, sig, params)
+		return CantoSigVerificationGasConsumer(meter, sig, params)
+	}
+}
+
+var _ authante.SignatureVerificationGasConsumer = CantoSigVerificationGasConsumer
+
+// SigVerificationGasConsumer is the canto implementation of SignatureVerificationGasConsumer. It consumes gas
+// for signature verification based upon the public key type. The cost is fetched from the given params and is matched
+// by the concrete type.
+// The types of keys supported are:
+//
+// - ethsecp256k1 (Ethereum keys)
+//
+// - ed25519 (Validators)
+//
+// - multisig (Cosmos SDK multisigs)
+func CantoSigVerificationGasConsumer(
```

Comment:
do we need this operation to consume gas? I mean the base tx fee is probably sufficient and I worry about correctness.